pyproject.toml

```diff
@@ -13,15 +13,15 @@ python = "^3.7.1"
 thrift = "^0.16.0"
 pandas = "^1.3.0"
 pyarrow = [
-    {version = ">=9.0.0", python = ">=3.7,<3.11"},
+    {version = ">=6.0.0", python = ">=3.7,<3.11"},
```

Comment:
Probably we could allow even older versions but I think this should make most people happy.